### PR TITLE
Add error state publisher

### PR DESCRIPTION
**Pull request by smits**

### CODE REVIEW

**Comment by adolfo-rt** [comment]
What's the context of this PR:
- Is this the state that we see in the logs when the camera goes beyond its max temperature, for instance?
- How do we expect to use this value?

**Review by adolfo-rt — APPROVED** [review]
Questions answered in sprint planning.

File: openni2_camera/src/openni2_device_manager.cpp [hunk 1]
```diff
@@ -67,6 +68,7 @@ class OpenNI2DeviceListener : public openni::OpenNI::DeviceConnectedListener,
       openni::OpenNI::DeviceDisconnectedListener(),
       openni::OpenNI::DeviceStateChangedListener()
   {
+    error_state_publisher_ = nh_.advertise<std_msgs::Int32>("error_state",100);
```
Comment (achim-k) [inline openni2_camera/src/openni2_device_manager.cpp]: This should be a latched topic:
```suggestion
    error_state_publisher_ = nh_.advertise<std_msgs::Int32>("error_state", 10, true);
```

**Review by achim-k — APPROVED** [review]
LGTM modulo comment. Changes should be binary compatible.

**Pull request merged**